lkmpg.tex

```diff
@@ -1086,7 +1086,7 @@ \subsection{Manage /proc file with seq\_file}
 The \cpp|seq_file| provides basic functions for \cpp|proc_ops|, such as \cpp|seq_read|, \cpp|seq_lseek|, and some others.
 But nothing to write in the \verb|/proc| file.
 Of course, you can still use the same way as in the previous example.
-
+\newpage
```

Comment:
I tend to avoid `\newpag` because we would re-organize the chapters later.

Reply:
ok, understood. i will remove this, thanks.
please wait.